h/static/scripts/annotator/guest.coffee

```diff
@@ -44,8 +44,14 @@ module.exports = class Guest extends Annotator
   html: extend {}, Annotator::html,
     adder: '''
       <div class="annotator-adder">
-        <button class="h-icon-insert-comment" data-action="comment" title="New Note"></button>
-        <button class="h-icon-border-color" data-action="highlight" title="Highlight"></button>
+        <div class="annotator-adder-action">
+          <button class="h-icon-annotate" data-action="comment"></button>
+          <label class="adder-label">Annotate</label>
```

Comment:
The label and the button elements are not connected here. You can either use the `for` attribute on the `<label>` or you could put the caption inside the `<button>` - have a look at the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) for details.